### PR TITLE
Use ISO dates in RFT ertobs files

### LINKED ISSUE #171: Update date format in `create_rft_ertobs` to YYYY-MM-DD
The old date format `DD MM YYYY` used in the RFT obs files has been deprecated by ERT, hence the `create_rft_ertobs` should be updated to write the new format `YYYY-MM-DD`

### PR DESCRIPTION
**Pull request by mferrera**
This PR updates `create_rft_ertobs` to write dates in ISO format, following #171 and the general initiative to move all dates to ISO format.

Resolves #171

### CODE REVIEW

**Review by berland — APPROVED** [review]
👍🏻 👍🏻

**Comment by mferrera** [comment]
I'm unable to merge this myself, if someone else wants to, or is willing to give me write access :-)

**Pull request merged**
